pkg/cli/cmd/find/find.go

```diff
@@ -68,129 +49,105 @@ func NewCmd(ctx context.DnoteCtx) *cobra.Command {
 	return cmd
 }
 
-// noteInfo is an information about the note to be printed on screen
-type noteInfo struct {
-	RowID     int
-	BookLabel string
-	Body      string
-}
-
-// formatFTSSnippet turns the matched snippet from a full text search
-// into a format suitable for CLI output
-func formatFTSSnippet(s string) (string, error) {
-	// first, strip all new lines
-	body := newLineReg.ReplaceAllString(s, " ")
-
-	var format, buf strings.Builder
-	var args []interface{}
-
-	toks := tokenize(body)
+func escapePhrase(s string) string {
+	escaped := strings.ReplaceAll(s, "%", "\\%")
+	escaped = strings.ReplaceAll(escaped, "_", "\\_")
 
-	for _, tok := range toks {
-		if tok.Kind == tokenKindHLBegin || tok.Kind == tokenKindEOL {
-			format.WriteString("%s")
-			args = append(args, buf.String())
-
-			buf.Reset()
-		} else if tok.Kind == tokenKindHLEnd {
-			format.WriteString("%s")
-			str := log.ColorYellow.Sprintf("%s", buf.String())
-			args = append(args, str)
-
-			buf.Reset()
-		} else {
-			if err := buf.WriteByte(tok.Value); err != nil {
-				return "", errors.Wrap(err, "building string")
-			}
-		}
-	}
-
-	return fmt.Sprintf(format.String(), args...), nil
-}
-
-// escapePhrase escapes the user-supplied FTS keywords by wrapping each term around
-// double quotations so that they are treated as 'strings' as defined by SQLite FTS5.
-func escapePhrase(s string) (string, error) {
-	var b strings.Builder
-
-	terms := strings.Fields(s)
-
-	for idx, term := range terms {
-		if _, err := b.WriteString(fmt.Sprintf("\"%s\"", term)); err != nil {
-			return "", errors.Wrap(err, "writing string to builder")
-		}
-
-		if idx != len(term)-1 {
-			if err := b.WriteByte(' '); err != nil {
-				return "", errors.Wrap(err, "writing space to builder")
-			}
-		}
-	}
-
-	return b.String(), nil
+	return "%" + escaped + "%"
 }
 
 func doQuery(ctx context.DnoteCtx, query, bookName string) (*sql.Rows, error) {
 	db := ctx.DB
 
-	sql := `SELECT
-		notes.rowid,
-		books.label AS book_label,
-		snippet(note_fts, 0, '<dnotehl>', '</dnotehl>', '...', 28)
-	FROM note_fts
-	INNER JOIN notes ON notes.rowid = note_fts.rowid
-	INNER JOIN books ON notes.book_uuid = books.uuid
-	WHERE note_fts MATCH ?`
+	sqlQuery := `SELECT
+            notes.rowid,
+            books.label AS book_label,
+            notes.body
+        FROM notes
+        JOIN books ON notes.book_uuid = books.uuid
+        WHERE notes.body LIKE ?`
```

Comment:
Hi @brobles82, this is an interesting idea. Is it correct to say that you are using `LIKE` instead of full-text search to match partial words?